src/components/TodoContainer.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import styles from "../styles/TodoContainer.module.css";
+import AddTodoForm from "./AddTodoForm";
+import Loading from "./Loading";
+import TodoList from "./TodoList";
+import { sortByField } from "../utils";
+import SortBox from "./SortBox";
+
+const LOCAL_STORAGE_SORT_KEY = "todoListSort";
+
+const BASE_URL = "https://api.airtable.com/v0";
+const BASE_ID = process.env.REACT_APP_AIRTABLE_BASE_ID;
+const TABLE_NAME = process.env.REACT_APP_TABLE_NAME;
+const API_TOKEN = process.env.REACT_APP_AIRTABLE_API_TOKEN;
+const GRID_VIEW = "view=Grid view";
+
+const AIRTABLE_URL = `${BASE_URL}/${BASE_ID}/${TABLE_NAME}`;
+
+const fetchAirtableData = async ({ method, url, body }) => {
+    const headers = {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${API_TOKEN}`
+    };
+
+    const axiosConfig = {
+        method: method,
+        url: `${AIRTABLE_URL}${url ?? ""}`,
+        headers: headers,
+        ...(body ? { data: body } : {})
+    };
+
+    try {
+        const response = await axios(axiosConfig);
+        return response.data;
+    } catch (error) {
+        throw new Error(`Error: ${error.response ? error.response.status : "Unknown"}`);
+    }
+};
```

Comment:
Recommendation:
Move this logic into an utility function. What does that mean? This means the logic is in a separate file under `src/utils/fetchAirtableData.js`. This will simplify your `App.js` file since this fetching data logic doesn't necessary tie directly to the component (it can be used anywhere).